<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subtitler</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="pCentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="pTabWidget">
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="pFileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="pOpenAction"/>
    <addaction name="pExitAction"/>
   </widget>
   <widget class="QMenu" name="pHelpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="pAboutQtAction"/>
    <addaction name="pAboutAction"/>
   </widget>
   <widget class="QMenu" name="pViewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="pPreviousPageAction"/>
    <addaction name="pNextPageAction"/>
    <addaction name="separator"/>
    <addaction name="pShowTimestampsAction"/>
   </widget>
   <addaction name="pFileMenu"/>
   <addaction name="pViewMenu"/>
   <addaction name="pHelpMenu"/>
  </widget>
  <widget class="QToolBar" name="pToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pOpenAction"/>
   <addaction name="separator"/>
   <addaction name="pPreviousPageAction"/>
   <addaction name="pNextPageAction"/>
   <addaction name="separator"/>
   <addaction name="pShowTimestampsAction"/>
  </widget>
  <widget class="QStatusBar" name="pStatusBar"/>
  <action name="pOpenAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open-file-icon</normaloff>:/icons/open-file-icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="pExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="pAboutQtAction">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="pAboutAction">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="pNextPageAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/next-page-icon</normaloff>:/icons/next-page-icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Page</string>
   </property>
   <property name="toolTip">
    <string>Scroll to Next Page</string>
   </property>
  </action>
  <action name="pPreviousPageAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/previous-page-icon</normaloff>:/icons/previous-page-icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous Page</string>
   </property>
   <property name="toolTip">
    <string>Scroll to Previous Page</string>
   </property>
  </action>
  <action name="pShowTimestampsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/show-timestamps-icon</normaloff>:/icons/show-timestamps-icon</iconset>
   </property>
   <property name="text">
    <string>Show Timestamps</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
